src/App.css

```diff
@@ -0,0 +1,79 @@
+@import url("https://fonts.googleapis.com/css2?family=Shadows+Into+Light&family=Staatliches&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 30px;
+}
+
+body,
+ul,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.App {
+  color: #ccc;
+  font-family: "Shizuru", cursive;
+  background-color: rgba(51, 51, 51, 0.97);
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.App__title {
+  font-family: "Shadows Into Light", cursive;
+  text-align: center;
+}
+
+.App__subtitle {
+  font-size: 1.5em;
+  font-style: italic;
+}
+
+.App__btn {
+  /* background-color: #bbb;
+   */
```

Comment:
do not forget to remove unused styles and comments

Reply:
sorry